docs/releasing/testing-and-linting.md

```diff
@@ -2,55 +2,88 @@
 
 The CI lints SASS and JavaScript, runs unit and functional tests with Node, and generates snapshots for [visual regression testing](https://www.browserstack.com/percy/visual-testing).
 
-## Running all tests locally
+## Testing terminology
+
+We use different types of tests to check different areas of our code are working as expected.
+
+Unit tests are small, modular tests that verify a "unit" of code. We write unit tests to check our JavaScript logic, particularly 'background logic' that does not heavily rely on [Document Object Model (DOM)](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction) interaction (where functional tests may be better suited).
+
+Functional tests verify the output of an action and do not check the intermediate states of the system when performing that action. We write functional tests to check component interactions have the expected results, so we also refer to these as component tests. We also write functional tests to check that our Nunjucks code outputs expected HTML, and we refer to these as our Nunjucks tests.
+
+[Snapshot tests](https://facebook.github.io/jest/docs/en/snapshot-testing.html) are used for preventing unintended changes to our component markup. When the snapshot test runs, it  compares the previously captured snapshot to the current markup.
+
+Visual regression tests help us check for any unintended visual changes to our components. We use [Percy](https://percy.io/) to generate and store screenshots of our components.
+
+## Running linting and tests
+
+### Running all tests locally
 
 To check the whole codebase, run:
 
 ```
 npm test
 ```
 
-This will trigger [standard](https://github.com/standard/standard) and [stylelint](https://github.com/stylelint/stylelint) for linting, and [Jest](https://github.com/facebook/jest) for unit and functional tests.
+This will trigger [standard](https://github.com/standard/standard) and [stylelint](https://github.com/stylelint/stylelint) for linting, and [Jest](https://github.com/facebook/jest) for our testing.
 
 See [Tasks](tasks.md) for details of what `npm test` does.
 
-## SASS linting
+### Running individual tests
+You can run a subset of the test suite that only tests components by running:
+
+    `npm test -- --testPathPattern=src/govuk/components/button`
```

Comment:
Interesting! I didn't know you could do it this way. I use `npx jest src/govuk/components/button` which _might_ be easier to remember 🤔

Reply:
Nice, didn't know about that 👍 I'll update this bit of the docs.